server/service/src/sync/sync_on_central/mod.rs

```diff
@@ -62,12 +62,10 @@ pub async fn pull(
         ));
     }
 
-    // Check credentials again mSupply central server
-    let response = SyncApiV5::new(sync_v5_settings)
-        .map_err(|e| Error::OtherServerError(format_error(&e)))?
-        .get_site_info()
+    let ctx = service_provider.basic_context()?;
+    let response = validate_site_auth(&ctx, &sync_v5_settings)
         .await
-        .map_err(Error::from)?;
+        .map_err(|e| Error::OtherServerError(format_error(&e)))?;
```

Comment:
Maybe this func should also return `Result<SyncBatchV6, CentralApiError>` rather than `Result<SyncBatchV6, SyncParsedErrorV6>`. Then it'd match `add_patient_name_store_join` returning `Result<_, CentralApiError>`, or vice versa. `validate_site_auth` can only do one or the other?

Reply:
Great point, can't wait till v7 where we don't need to stack errors like this : ), a bit hard to know why it was done as CentralApiError now ¯\_(ツ)_/¯